lib/Twig/Extension/Core.php

```diff
@@ -1634,4 +1635,31 @@ function twig_get_attribute(Twig_Environment $env, Twig_Source $source, $object,
     return $ret;
 }
 
+function twig_classes(...$args): string
+{
+    $classes = [];
+
+    foreach ($args as $i => $arg) {
+        if (is_string($arg)) {
```

Comment:
should we allow a `Twig_Markup` object here, by casting it ?

Reply:
I guess yes :) I will update the PR

Reply:
@stof I added a test for that. I did not had to update the code :+1: